course/addquestions.php

```diff
@@ -144,11 +153,31 @@
 			require_once('../includes/filehandler.php');
 			deleteallaidfiles($aid);
 			if ($aver > 1) {
+                $stm = $DBH->query("SELECT userid,score FROM imas_assessment_records WHERE assessmentid=:assessmentid");

```

Comment:
This is an invalid query: needs to use `prepare` with an execute if using a placeholder